base/initdefs.jl

```diff
@@ -42,6 +42,7 @@ function init_load_path()
     end
     push!(LOAD_PATH, abspath(JULIA_HOME, "..", "local", "share", "julia", "site", vers))
     push!(LOAD_PATH, abspath(JULIA_HOME, "..", "share", "julia", "site", vers))
+    push!(LOAD_PATH, abspath(JULIA_HOME, "..", "..", "stdlib"))
```

Comment:
Yeah, that's probably a good idea; even though it's not totally obvious that it's the julia `stdlib`, it makes more sense because long-term we're planning on moving things out of this repo anyway.